app/core/src/main/java/com/thechance/core/data/usecase/user/UserUseCaseContainer.kt

```diff
@@ -0,0 +1,7 @@
+package com.thechance.core.data.usecase.user
+
+import org.koin.core.component.KoinComponent
+
+data class UserUseCaseContainer(
```

Comment:
Not neccessary to make container here you just has one useCase

Reply:
But later won't we add delete User and  update?